test/fit_tests/deploy/rackhd_stack_init.py

```diff
@@ -19,7 +30,7 @@
 # Locals
 MAX_CYCLES = 60
 
-class rackhd_stack_init(fit_common.unittest.TestCase):
+class rackhd_stack_init(unittest.TestCase):
     def test01_preload_sku_packs(self):
         print "**** Processing SKU Packs"
         # Load SKU packs from GutHub
```

Comment:
I tested updates to this, skipping the temp directory creation and works in my environment.  However, after talking to George on why it was implemented with this temp dir, not sure if it works in all environments (running it from Jenkins), so for now, going to leave it as is.